supertokens_python/recipe/emailverification/emaildelivery/service/backwardCompatibility/__init__.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
change the directory from backwardCompatibility to backward_compatibility

Reply:
Done. Thanks!